README.md

```diff
@@ -27,13 +24,11 @@ Additional description about the project and its features.
 
 To get a local copy up and running follow these simple example steps.
 
-### Prerequisites
+### Prerequisites - Text editor | Git and Github setup
 
-### Setup
+### Setup - clone repository and open with text editor
 
-### Install
-
-### Usage
+### Usage - View Hello World
```

Comment:
- [ ] To add a section description in `README` you have to start the description from  new line. Please try:

```
### Prerequisites 
- Text editor | Git and Github setup

### Setup 
- clone repository and open with text editor
```
Feel free to follow the [link](https://docs.github.com/en/github/writing-on-github/getting-started-with-writing-and-formatting-on-github/basic-writing-and-formatting-syntax) and learn about basic writing and formatting syntax to make `README` look great 💯

Reply:
Thank you! You were very constructive and helpful with your comments.